src/lib/ngsiNotify/QueueWorkers.cpp

```diff
@@ -103,6 +106,14 @@ static void *workerFunc(void* pSyncQ)
     {
       statisticsUpdate(NotifyContextSent, params->format);
     }
+    if (r != "error" && r != "")
+    {
+      ++QueueStatistics::noOfNotificationsQueueOK;
```

Comment:
I'd suggest noOfNotificationsQueueSentOK. 'Sent' is important to mean that an actual HTTP interaction take place (i.e. it was not only movements in the internal queue).

In addition, probably this SendOk and SendError counters should be general for all notification modes. Please, add a FIXME about it around this code.